Python/Product/PythonTools/PythonTools/LanguageServerClient/PythonLanguageClient.cs

```diff
@@ -475,7 +475,18 @@ private void OnAnalysisComplete(object sender, EventArgs e) {
         private void OnReanalyzeChanged(object sender, EventArgs e) {
             try {
                 Debug.WriteLine("Reanalyze Changed");
-                _deferredSettingsChangedTimer.Change(_defaultSettingsDelayMS, Timeout.Infinite);
+
+                // Packages have changes, so send a workspace/didChangeWatchedFiles for the env directory.
+                this._clientContexts.ForEach(context => {
+                    var createEvent = new FileEvent();
+                    createEvent.FileChangeType = FileChangeType.Changed;
+                    createEvent.Uri = new Uri(CommonUtils.GetParent(context.InterpreterConfiguration.InterpreterPath));
+                    var didChangeParams = new DidChangeWatchedFilesParams();
+                    didChangeParams.Changes = new FileEvent[] { createEvent };
+                    _rpc.NotifyWithParameterObjectAsync(Methods.WorkspaceDidChangeWatchedFiles.Name, didChangeParams);
+                });
```

Comment:
on a package added/removed event this 'OnReanalyzeChanged' handler is called.  Now we loop through our clientcontexts and send the directory of the interpreterPath ie. 'python.exe' directory.

Reply:
yes triggered by. 'PackageManager_InstalledFilesChanged'